cmd/migrate/down.go

```diff
@@ -35,6 +37,11 @@ func newDownCmd() *cobra.Command {
 			}
 			cmdx.PrintTable(cmd, s)
 
+			if !flagx.MustGetBool(cmd, FlagYes) || !cmdx.AskForConfirmation("Do you really want to migrate down? This will delete data.", cmd.InOrStdin(), cmd.OutOrStdout()) {
```

Comment:
```suggestion
			if !flagx.MustGetBool(cmd, FlagYes) || !cmdx.AskForConfirmation("Do you really want to migrate down? This will irrecoverably delete data.", cmd.InOrStdin(), cmd.OutOrStdout()) {
```